common/models/create_environment_request.go

```diff
@@ -21,16 +22,13 @@ func (r CreateEnvironmentRequest) Validate() error {
 		return fmt.Errorf("EnvironmentName is required")
 	}
 
-	if r.MinScale < 0 {
-		return fmt.Errorf("MinScale must be a positive integer")
+	if r.Scale < 0 {
+		return fmt.Errorf("Scale must be a positive integer")
 	}
 
-	if r.MaxScale < 0 {
-		return fmt.Errorf("MaxScale must be a positive integer")
-	}
-
-	if r.MaxScale < r.MinScale {
-		return fmt.Errorf("MaxScale must be greater than or equal to MinScale")
+	if strings.EqualFold(r.EnvironmentType, EnvironmentTypeDynamic) &&
```

Comment:
I'm not sure why we are using `strings.EqualFold` here; why isn't default comparison (`==`) sufficient?

Reply:
for a case insensitive comparison.